### PR TITLE
Update installation-overview.md

### PR DESCRIPTION
**Pull request by runleonarun**
## What are you changing in this pull request and why?
This PR contains info about  upgrading dbt Core per [this PR ](https://github.com/dbt-labs/docs.getdbt.com/pull/977)opened by @jasnonaz.

## Checklist

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] peer review

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

**Review by mirnawong1 — APPROVED** [review]
hey @runleonarun approving this for you but also wanted to flag a nonblocking question- ask how come we're only referencing pip upgrades and not others ([homebrew](/docs/get-started/homebrew-install#upgrading-dbt-and-your-adapter)), for example. or our [about core versions]/docs/dbt-versions/core) doc.

**Comment by runleonarun** [comment]
@mirnawong1 this is a great callout and probably because the original PR was old and those weren't up yet. Before we merge this PR, we should:
- Add all Core upgrade options or just point to  our [about core versions]/docs/dbt-versions/core) per Mirna's suggestion!

File: website/docs/docs/get-started/installation-overview.md [hunk 1]
```diff
@@ -11,7 +11,10 @@ You can install dbt Core on the command line by using one of these recommended m
 - [Use a Docker image to install dbt](/docs/get-started/docker-install)
 - [Install dbt from source](/docs/get-started/source-install)
 
+## Upgrading dbt Core
 
-## About dbt adapters
+To upgrade dbt using `pip`, see the [pip upgrading instructions](/docs/get-started/pip-install#change-dbt-core-versions). dbt Labs provides a number of resources for understanding [general best practices](/blog/upgrade-dbt-without-fear) while upgrading your dbt project as well as detailed [migration guides](/guides/migration/versions/upgrading-to-v1.4) highlighting the changes required for each minor and major release.
```
Comment (mirnawong1) [inline website/docs/docs/get-started/installation-overview.md]:
```suggestion
dbt provides a number of resources for understanding [general best practices](/blog/upgrade-dbt-without-fear) while upgrading your dbt project as well as detailed [migration guides](/guides/migration/versions/upgrading-to-v1.4) highlighting the changes required for each minor and major release, and [core versions](/docs/dbt-versions/core)

- [Upgrade Homebrew](/docs/get-started/homebrew-install#upgrading-dbt-and-your-adapter)
- [Upgrade `pip`](/docs/get-started/pip-install#change-dbt-core-versions)

```

**Review by mirnawong1 — DISMISSED** [review]
hey @runleonarun made this tweak and shipping it for you - let me know if that works!

**Review by mirnawong1 — APPROVED** [review]

**Pull request merged**